src/sst/core/elemLoader.h

```diff
@@ -40,9 +35,17 @@ class ElemLoader
     void loadLibrary(const std::string& elemlib, std::ostream& err_os);
 
     /**
-     * Returns a list of potential element libraries in the search path
+     * Search paths for potential elements and add them to the provided vector
+     *
+     * @param potElems - vector of potential elements that could contain elements
+     * @return void
      */
-    std::vector<std::string> getPotentialElements();
+    void getPotentialElements(std::vector<std::string>& potElems);
+
+private:
+    std::string searchPaths;
+    bool        verbose;
+    int         bindPolicy;
```

Comment:
Set `int bindPolicy = RTLD_LAZY | RTLD_GLOBAL;` here or (in the constructor if you don't want to pull in the header with these symbols right here).

Reply:
See above.